src/hyperloglog.c

```diff
@@ -1772,16 +1958,22 @@ void pfdebugCommand(client *c) {
         if (!strcasecmp(c->argv[2]->ptr, "on")) {
 #ifdef HAVE_AVX2
             simd_enabled = 1;
+#endif
+#ifdef __ARM_NEON
+            simd_enabled = 1;
 #endif
```

Comment:
#if defined(HAVE_AVX) || defined(__ARM_NEON)

or we can consider some macro like `HLL_HAVE_SIMD` to avoid this repetition, also to avoid defining the variable twice.